SPECS/cmake/cmake.spec

```diff
@@ -11,9 +11,7 @@ URL:            https://www.cmake.org/
 Source0:        https://github.com/Kitware/CMake/releases/download/v%{version}/%{name}-%{version}.tar.gz
 Source1:        macros.cmake
 Patch0:         disableUnstableUT.patch
-# Please note that this is an empty patch file whose only purpose is to serve as a placeholder to notify
-# the CVE scanning tool that the CVE is resolved. The real fix for the vulnerability is to add --system-curl
-# to the bootstrap phase down below
+# We could use --system-curl instead of patching, but unfortuately curl isn't currently available in time during the toolchain build.
```

Comment:
Please file a bug and assign it to me, this looks like a fairly straightforward fix but no need to block the PR on it. I'll be working on a couple of toolchain fixes this week anyways.